specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2021-11-01-preview/examples/AgentPoolsAssociate_CRG.json

```diff
@@ -11,8 +11,7 @@
         "count": 3,
         "vmSize": "Standard_DS2_v2",
         "osType": "Linux",
-        "capacityReservationGroupID": "/subscriptions/subid1/resourcegroups/rg1/providers//Microsoft.Compute/CapacityReservationGroups/crg1"
-      }
+        "capacityReservationGroupResourceId": "/subscriptions/subid1/resourcegroups/rg1/providers//Microsoft.Compute/CapacityReservationGroups/crg1"
```

Comment:
Looks like we are missing the `}` here

Reply:
From the comparison, I don't remove any } here. The indent level of the request is different from the response's indent level.

Reply:
Interesting, after removing some filters, I can one } is missed. But in my IDE, I don't see any } missing.